views/settings/fields/categorized_multiselect.php

```diff
@@ -0,0 +1,204 @@
+	?>
+	<li>
+		<div class="wpr-checkbox">
+			<input type="checkbox" name="<?php echo esc_attr( $id ); ?>"
+				value='<?php echo esc_attr( wp_json_encode( $item->exclusions ) ); ?>'
+				<?php echo in_array( $id, $state, true ) ? ' checked="checked"' : ''; ?> />
+			<label> <img src="<?php echo esc_url( fetch_icon( $item ) ); ?>"/>
+			<?php echo esc_attr( $item->title ); ?>
+			</label>
+		</div>
+	</li>
+	<?php
+}
+
+/**
+ * Render the list.
+ *
+ * @param string   $title title from the list.
+ * @param string   $input_name name from the input.
+ * @param stdClass $list list to render.
+ * @param array    $state current state.
+ * @param bool     $open is the list open.
+ * @return void
+ */
+function render_list( string $title, string $input_name, stdClass $list, array $state, bool $open = false ) {
+	$has_selected = false;
+
+	if ( count( (array) $list ) === 0 ) {
+		return;
+	}
+
+	foreach ( $list as $id => $item ) {
+		if ( in_array( $id, $state, true ) ) {
+			$has_selected = true;
+		}
+	}
+	?>
+		<div class="wpr-list<?php echo $open ? ' open' : ''; ?>">
+			<div class="wpr-list-header">
+				<div class="wpr-checkbox">
+					<input class="wpr-main-checkbox" type="checkbox" id="<?php echo esc_attr( $input_name ); ?>" name="<?php echo esc_attr( $input_name ); ?>"
+											<?php
+											echo $has_selected ? ' checked="checked"'
+											: '';
+											?>
+							/>
+
+					<label><span class="wpr-multiple-select-<?php echo esc_attr( $input_name ); ?>">
+							<?php
+								echo esc_attr( $title );
+							?>
+					</span></label>
+				</div>
+				<div class="wpr-list-header-arrow">
+					<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. --><path d="M233.4 105.4c12.5-12.5 32.8-12.5 45.3 0l192 192c12.5 12.5 12.5 32.8 0 45.3s-32.8 12.5-45.3 0L256 173.3 86.6 342.6c-12.5 12.5-32.8 12.5-45.3 0s-12.5-32.8 0-45.3l192-192z"/></svg>
+				</div>
+			</div>
+			<div class="wpr-list-body">
+				<ul>
+					<?php
+					foreach ( $list as $id => $item ) :
+						render_list_item( $id, $item, $state );
+					endforeach;
+					?>
+				</ul>
+			</div>
+		</div>
+	<?php
+}
+
+?>
+
+<div id = '<?php echo esc_attr( $data['id'] ); ?>' class="wpr-field wpr-multiple-select <?php echo esc_attr( $data['container_class'] ); ?>" data-default="<?php echo ( ! empty( $data['default'] ) ? 'wpr-radio-' . esc_attr( $data['default'] ) : '' ); ?>" <?php echo $data['parent']; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- $data['parent'] escaped with esc_attr. ?>>
+
+	<h2><?php echo esc_html( __( 'Excluded JavaScript Files', 'rocket' ) ); ?></h2>
+	<p>
+	<?php
+	echo esc_html( __( 'Add JavaScript files to be excluded from delaying execution by selected them / or by add in “My scripts”.', 'rocket' ) );
```

Comment:
`My scripts` looks strange I think we need to change it to `delayed scripts` or find a better title

Reply:
@GeekPress to be fair I agree with @mostafa-hisham it took me time to understand what was `My scripts`